language/stdlib/modules/LibraSystem.move

```diff
@@ -1,7 +1,9 @@
 address 0x1 {
 
-/// The `LibraSystem` module provides an interface for maintaining information
-/// about the set of validators used during consensus.
+/// Maintains information about the set of validators used during consensus.
+/// Provides functions to add, remove, and update validators in the
+/// validator set.
+/// > Note: The terms "configuration" and "config" are heavily overloaded in this code.
```

Comment:
I think its fine to use the full spelling and shortcuts. Let's perhaps remove? I don't mind that much, but it sounds like self-criticism.